config/ehcache.xml

```diff
@@ -372,106 +356,31 @@
     -->
 
     <!--
-     The data stored in this cache is relatively small and have a short idle time.
-     THIS CACHE IS DISTRIBUTED.
+     The data store in this cache is large and are meant to not exceed a specified max memory allocation.
+     This cache is shared among all web applications in the container.  it will NOT be replicated.
+     The maxBytesLocalHeap is not important here because we will automatically calculate it based on
+     pct.vis.shared.mem.size property.
     -->
-    <cache name="HTTP_SESSION"
-           maxElementsInMemory="200"
+    <cache name="VIS_SHARED_MEM"
+           maxBytesLocalHeap="100M"
```

Comment:
we override 100M somewhere, right?

Reply:
Yes.  We override this setting at runtime based on JVM size.